apps/website/src/components/Placeholder.astro

```diff
@@ -0,0 +1,39 @@
+---
+import { SvgInfoCircular } from '@itwin/itwinui-icons-react';
+
+export type Props = {
+  /** ending of the url for a specific storybook page */
+  componentPageName: string;
+};
+
+const { componentPageName } = Astro.props;
+---
+
+<div class='alert' data-iui-status='informational'>
+  <SvgInfoCircular className='iui-alert-icon' aria-hidden='true' />
+  <span class='iui-alert-message'
```

Comment:
we are trying to maintain a distinct design language for the website, so let's not use anything from iTwinUI (no `iui-` classes / itwinui-icons).

you don't need to make anything fancy, but would be nice if you can use spacing/color values from `__global.astro` (line 116 onwards).

for icons, you can grab something from https://icones.js.org/collection/all and either add it to the `website/src/icons/` folder or inline it in CSS.

Reply:
here is an example implementation you can build upon:
```astro
---
export type Props = {
  /** ending of the url for a specific storybook page */
  componentPageName: string;
};

const { componentPageName, ...rest } = Astro.props;

const storyUrl = `https://itwin.github.io/iTwinUI/react/?path=/story/${componentPageName}`;
---

<div {...rest}>
  <p>
    This page has not yet been finished and is being worked on. In the meantime, please visit
    <a href={storyUrl}>storybook</a> to see more examples.
  </p>
</div>

<style lang='scss'>
  div {
    border-block: 1px solid var(--color-line-1);
    padding-block: var(--space-3);
    display: flex;
    align-items: center;
    gap: var(--space-2);

    &::before {
      content: '';
      flex-shrink: 0;
      display: inline-block;
      width: 1.5em;
      height: 1.5em;

      --_svg: url('data:image/svg+xml,<svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 24 24"><path fill="currentColor" d="M13 9a1 1 0 1 1-2 0a1 1 0 0 1 2 0Zm-1 2.75a.75.75 0 0 1 .75.75v5a.75.75 0 1 1-1.5 0v-5a.75.75 0 0 1 .75-.75Z"/><path fill="currentColor" fill-rule="evenodd" d="M14.27 3.993a2.749 2.749 0 0 0-4.54 0l-.432.632a75.95 75.95 0 0 0-6.944 12.563l-.09.208a2.511 2.511 0 0 0 2.024 3.497a69.43 69.43 0 0 0 15.424 0a2.511 2.511 0 0 0 2.024-3.497l-.09-.208a75.951 75.951 0 0 0-6.944-12.563l-.432-.632Zm-3.302.846a1.25 1.25 0 0 1 2.064 0l.432.632a74.444 74.444 0 0 1 6.806 12.315l.09.208a1.011 1.011 0 0 1-.814 1.408c-5.015.56-10.077.56-15.092 0a1.011 1.011 0 0 1-.815-1.408l.09-.208a74.45 74.45 0 0 1 6.807-12.315l.432-.632Z" clip-rule="evenodd"/></svg>');
      -webkit-mask-image: var(--_svg);
      mask: var(--_svg) no-repeat center / contain;
      background-color: currentColor;
    }
  }
</style>

```

Reply:
Updated